ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-authentication.j2

```diff
@@ -10,4 +10,17 @@ aaa authentication login serial-console {{ aaa_authentication.login.serial_conso
 {%     if aaa_authentication.dot1x.default is defined and aaa_authentication.dot1x.default is not none %}
 aaa authentication dot1x default {{ aaa_authentication.dot1x.default }}
 {%     endif %}
+{%     if aaa_authentication.policies is defined and aaa_authentication.policies is not none %}
+{%         if aaa_authentication.policies.on_failure_log is defined and aaa_authentication.policies.on_failure_log == true %}
+aaa authentication policy on-failure log
+{%         endif %}
+{%         if aaa_authentication.policies.on_success_log is defined and aaa_authentication.policies.on_success_log == true %}
+aaa authentication policy on-success log
+{%         endif %}
+{%         if aaa_authentication.policies.local is defined %}
+{%             if aaa_authentication.policies.local.allow_nopassword is defined and aaa_authentication.policies.local.allow_nopassword == false %}
```

Comment:
EOS Default is that this is _not_ configured. Logic must be reversed.

Reply:
Resolved.